common/domain-utils.ts

```diff
@@ -62,7 +62,8 @@ export const parseSettings = (setting: string): PageSetting => {
     const [landingPageEncoded, mode, ...extensions] = segment(setting)
     const landingPage = decodeURIComponent(landingPageEncoded)
     const unrestrictedMode = mode !== 'strict'
-    const farcastEnabled = extensions.includes('farcast')
+    const farcastPartial = extensions.includes('farcast-partial')
```

Comment:
note: I will need to change some demo site's configurations, because this change is not backward compatible